src/test/regress/pg_regress_multi.pl

```diff
@@ -115,10 +115,13 @@ ()
 # want to find binaries if they're not in PATH.
 if (defined $bindir)
 {
-    $ENV{PATH} = ($ENV{PATH} || '').":$bindir";
+    if ($usingWindows) {
+        $ENV{PATH} = ($ENV{PATH} || '').";$bindir";
```

Comment:
1. What about `$libdir`? I think we should also use semi-colon separators for VS there?
2. Does `LD_LIBRARY_PATH` work for VS? [MSDN](https://msdn.microsoft.com/en-us/library/6y6t9esh.aspx) says it is `LINK`?

Reply:
Pushed a change which correctly adds `$libdir` to `PATH` if you've set it. None of the other variables are relevant for Windows. Looks like Windows uses both `LIB` and `LINK`. I don't know why someone would want to set those though, on Windows installation works by adding everything you could need into `$prefix/lib`, for better or worse.